TaskHouseApi/Persistence/DatabaseContext/Seed.cs

```diff
@@ -19,6 +19,91 @@ public static void Initialize(IServiceProvider serviceProvider)
                     return;   // DB has been seeded
                 }
 
+                context.Locations.AddRange
+               (
+                   new Location()
+                   {
+                       Country = "Country1",
+                       City = "City1",
+                       ZipCode = "ZipCode1",
+                       PrimaryLine = "PrimaryLine1",
+                       SecondaryLine = "SecondaryLine1"
+                   },
+                   new Location()
+                   {
+                       Country = "Country2",
+                       City = "City2",
+                       ZipCode = "ZipCode2",
+                       PrimaryLine = "PrimaryLine2",
+                       SecondaryLine = "SecondaryLine2"
+                   },
+                   new Location()
+                   {
+                       Country = "Country3",
+                       City = "City3",
+                       ZipCode = "ZipCode3",
+                       PrimaryLine = "PrimaryLine3",
+                       SecondaryLine = "SecondaryLine3"
+                   },
+                    new Location()
```

Comment:
have identical seedata could  be confusing couldn't it? if i asume i will get 1 elemet back and gets 3 coun't that be mistaken for an error?

Reply:
True - but I guess it wont be a problem since it is only locations

Reply:
i like this responds